src/tasks/Common/FileCache.cs

```diff
@@ -34,7 +35,7 @@ public FileCache(string? cacheFilePath, TaskLoggingHelper log)
         {
             _oldCache = (CompilerCache?)JsonSerializer.Deserialize(File.ReadAllText(cacheFilePath),
                                                                     typeof(CompilerCache),
-                                                                    new JsonSerializerOptions());
+                                                                    s_jsonOptions);
```

Comment:
Does this need to pass a JSO at all? Couldn't the argument just be removed?

Reply:
I think it's better for consistency to keep it here. In case another option is added to the options in the future apart from WriteIndented, someone might not realize that now they have to pass in the options here as well.

Reply:
The options just happen to not affect reading right now, but that could change.

Reply:
Also, this way, the same cache is reused for both reading & writing.